### PR TITLE
add sysbench workload

### PR DESCRIPTION
**Pull request by BugenZhao**

### CODE REVIEW

File: bench/record/record.bash [hunk 1]
```diff
@@ -15,6 +15,10 @@ bench_start=`env_val "${env}" 'bench.start'`
 if [ -z "${bench_start}" ]; then
 	bench_start='0'
 fi
+test_name=`env_val "${env}" "bench.${workload}.test-name"`
+if [ ! -z "${test_name}" ]; then
+	workload=${workload}-${test_name}
```
Comment (innerr) [inline bench/record/record.bash:20]: workload has a specific meaning, suggest:
1, put test_name to tag(s) in score table
2, use a spare col for test_name (alter table to add a col) in detail table

That's what detail table for.

File: bench/record/record.bash [hunk 2]
```diff
@@ -15,6 +15,10 @@ bench_start=`env_val "${env}" 'bench.start'`
 if [ -z "${bench_start}" ]; then
 	bench_start='0'
 fi
+test_name=`env_val "${env}" "bench.${workload}.test-name"`
```
Comment (innerr) [inline bench/record/record.bash:18]: This module `bench.record` is for all workloads, other workload may not have `test-name` but have other props.

The best way is use score table for common info, then use detail table to record as many info as posible.

File: sysbench/load.bash [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+tables=`must_env_val "${env}" 'bench.sysbench.tables'`
+table_size=`must_env_val "${env}" 'bench.sysbench.table-size'`
+test_name=`must_env_val "${env}" 'bench.sysbench.test-name'`
+
+host=`must_env_val "${env}" 'mysql.host'`
+port=`must_env_val "${env}" 'mysql.port'`
+user=`must_env_val "${env}" 'mysql.user'`
+db=sbtest
+
+mysql -h "${host}" -P "${port}" -u "${user}" -e "set global tidb_disable_txn_auto_retry = off"
```
Comment (innerr) [inline sysbench/load.bash:16]: Use upper chars for `set` `global` ...

File: sysbench/run.bash.ticat [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+bench.sysbench.test-name|name = read
+bench.sysbench.duration|dur|d|D|time = read
+mysql.host = read
+mysql.port = read
+mysql.user = read
+bench.run.log = write
+bench.run.score = write
+
+[dep]
+sysbench = to run the bench
```
Comment (innerr) [inline sysbench/run.bash.ticat:36]: Consider this: add sysbench as git-sub-module under <this-repo>/repos, then built it if not have sysbench installed.
(have did that in the old version tool `ti.sh`, it runs better with auto installing)

Comment (innerr) [reply sysbench/run.bash.ticat:36]: Or not adding sysbench as git-sub-module, only pull it down to a temp dir when it's needed

File: bench/record/record.bash (hunk 1 above)
Comment (BugenZhao) [reply bench/record/record.bash:20]: `test_name` is already encoded in the `tag`, so I will just remove these lines.

File: sysbench/run.bash.ticat (hunk 4 above)
Comment (BugenZhao) [reply sysbench/run.bash.ticat]: We may currently assume the users are on popular distributions and have a known package manager installed, then we can easily install the missing dependencies from it.

**Review by innerr — APPROVED** [review]

**Pull request merged**